back/app/services/survey_results_generator_service.rb

```diff
@@ -100,7 +100,7 @@ def core_field_attributes(field, response_count)
       question: field.title_multiloc,
       customFieldId: field.id,
       required: field.required,
-      grouped: false,
+      grouped: !!group_field_id,
```

Comment:
Why do you need to change it? Isn't it always overwritten with `grouped: !!group_field,` when it can be `true`?

Reply:
I mean we don't need to check it two times, do we?

Reply:
Indeed. Removed the duplication 👍